app/services/stock/index.js

```diff
@@ -23,4 +21,6 @@ const connectDB = async () => {
 
 connectDB();
 
+app.use('/equity', stockRoute);
```

Comment:
I used the `/equity` endpoint for stock prices but we merge the two together into one routes file and just use both controllers.